pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -204,6 +202,20 @@ func (r *ReconcileNode) createOrUpdateCephCron(cephCluster cephv1.CephCluster, c
 	return controllerutil.CreateOrUpdate(r.opManagerContext, r.client, cronJob, mutateFunc)
 }
 
+func (r *ReconcileNode) deletev1betaAndCreatev1Job(objectMeta metav1.ObjectMeta) error {
```

Comment:
It doesn't create the job, right?
```suggestion
func (r *ReconcileNode) deletev1betaJob(objectMeta metav1.ObjectMeta) error {
```

Reply:
right